tools/build.py

```diff
@@ -244,11 +248,19 @@ def configure_jerry(arguments):
 
     return subprocess.call(cmake_cmd)
 
-def make_jerry(arguments, target=None):
-    make_cmd = ['make', '--no-print-directory', '-j', str(arguments.jobs), '-C', arguments.builddir]
+def make_jerry(arguments):
+    if sys.platform == 'win32':
+        make_cmd = ['cmake', '--build', arguments.builddir, '--config', arguments.build_type]
+    else:
+        make_cmd = ['make', '--no-print-directory', '-j', str(arguments.jobs), '-C', arguments.builddir]
```

Comment:
I'd say we should build/install with cmake calls on all platforms. What do you think?

Reply:
It sounds good, I'll check if this idea works on other platforms too. I think it should, but let's try it.

Reply:
Done in the latest updated commit.